common/k8s/src/middleware/metadata.rs

```diff
@@ -9,12 +9,17 @@ use kube::{api::ListParams, config::Config, Api, Client};
 use kube_runtime::watcher;
 use kube_runtime::watcher::Event as WatcherEvent;
 
+use backoff::backoff::Backoff;
```

Comment:
Nice, replacing out custom "limit" implementation in common/http with this lib is on my TODO list